src/dotnet/Microsoft.FxLister/README.md

```diff
@@ -0,0 +1,108 @@
+# Microsoft.FxLister
```

Comment:
Please add a CODEOWNERS entry for this tool and add @KrzysztofCwalina as the owner.